www-client/pybugz/pybugz-0.13-r3.ebuild

```
@@ -0,0 +1,32 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
```

let's drop NOP target 3.9

```suggestion
PYTHON_COMPAT=( python3_{10..11} )
```

Also, do you know if 3.12 works?

---

No issue in such case to skip 3.12 check/addition and leave upstream / maintainer.